src/med_bench/get_estimation.py

```diff
@@ -676,8 +623,7 @@ def get_estimation(x, t, m, y, estimator, config):
                 crossfit=2,
                 clip=0.01,
                 regularization=True,
-                calibration=True,
-                calib_method="isotonic",
+                calibration_method="isotonic",
```

Comment:
```suggestion
                calibration="isotonic",
```

Reply:
This has been addressed in commit 14374c26028c5a15b5ce4dd3e8cd907a83d83a37